modules/nix/nix-darwin.nix

```diff
@@ -23,7 +34,7 @@ in
 
   config = {
     environment.systemPackages =
-      [ # Include nix-tools by default
+      lib.optionals (!config.system.disableInstallerTools) [
         darwin-option
         darwin-rebuild
         darwin-version
```

Comment:
It seems to me that you would probably still want `darwin-version`

Reply:
thats a fair point